### DISCUSSION TITLE
金狼，银狼，铁狼，，

### DISCUSSION THREAD

**Discussion by dengyunzhe** [share121/inter-knot#1347] [其他]
![嘿嘿，银狼(¯﹃¯)](https://i0.hdslb.com/bfs/article/8881b415ed29a4ccca8758ab8217cc9c298252208.png@1256w_806h_!web-article-pic.avif)

![银狼の披萨](https://i0.hdslb.com/bfs/article/8b153b1a883f8f4417b14784140b5338298252208.jpg@1256w_1778h_!web-article-pic.avif)

![1](https://i0.hdslb.com/bfs/article/8c52f0ccb3d279dc5d842898990b858b298252208.png@1256w_1906h_!web-article-pic.avif)

![杂鱼❤视角](https://i0.hdslb.com/bfs/article/736f81d08325f11cc2274c7446b4febe298252208.png@1256w_2430h_!web-article-pic.avif)

![雨一直下](https://act-upload.mihoyo.com/sr-wiki/2023/12/05/279865110/7b990e147806e52fff7fdd36761adc50_4560410625005761595.png)

![玉足](https://i0.hdslb.com/bfs/article/6da936c8d1243cb61d1babc14c23855e298252208.jpg@1256w_1890h_!web-article-pic.avif)

**Comment by Sunny-Ding110** [comment]
这个我喜欢

**Comment by XxnjiIAJ** [comment]
豪赤！
